src/alto-ui/Datagrid/Datagrid.js

```diff
@@ -123,6 +127,10 @@ class Datagrid extends React.PureComponent {
     this.trackDimensions();
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.trackDimensions);
```

Comment:
this will not work and you will have a memory leak here because it is not this function that has been added as a listener but a debounced version of it.